src/components/DetailsInfo/DetailsInfoView.js

```diff
@@ -93,6 +93,10 @@ const DetailsInfoView = React.forwardRef(
                       ? changes.data[header.id]
                         ? changes.data[header.id]
                         : parseKeyValues(selectedItem.labels)
+                      : content[header.id]?.value === selectedItem.metrics
+                      ? changes.data[header.id]
```

Comment:
we don't have edit mode for metrics, so there no need to check if changes array has some changed metrics